### ISSUE TITLE
Version 0.3.3 logs out at `ERROR` level

### ISSUE THREAD

**Issue by w25r** [kxbmap/sbt-jooq#13]
Version 0.3.2 SBT output
```console
INFO: 
                                      
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@  @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@
@@@@@@@@@@@@@@@@  @@  @@    @@@@@@@@@@
@@@@@@@@@@  @@@@  @@  @@    @@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
@@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
@@@@@@@@@@        @@  @  @  @@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@  @@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  Thank you for using jOOQ 3.9.4
```
Version 0.3.3 SBT output
```console
[error] INFO: 
[error]                                       
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[error] @@@@@@@@@@@@@@@@  @@        @@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@
[error] @@@@@@@@@@@@@@@@  @@  @@    @@@@@@@@@@
[error] @@@@@@@@@@  @@@@  @@  @@    @@@@@@@@@@
[error] @@@@@@@@@@        @@        @@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[error] @@@@@@@@@@        @@        @@@@@@@@@@
[error] @@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
[error] @@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
[error] @@@@@@@@@@        @@  @  @  @@@@@@@@@@
[error] @@@@@@@@@@        @@        @@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@  @@@@@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[error] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  Thank you for using jOOQ 3.9.4
```

Note, both of the above tests were using sbt.version=1.1.2

**Comment by kxbmap** [comment]
jOOQ logo is printed by jOOQ library using one of the logger APIs (please refer [JooqLogger javadoc](https://www.jooq.org/javadoc/3.12.x/org.jooq/org/jooq/tools/JooqLogger.html)).
You can control log output by logging framework you use.

**Comment by w25r** [comment]
@kxbmap you can control what output shows up... but it is now being logged at `ERROR`.  The logging framework is immaterial.  And it's not just the logo, it's all jOOQ's output.

**Comment by kxbmap** [comment]
When did this log show? at `jooqCodegen` or at `run`?

**Comment by w25r** [comment]
`jooqCodegen`

**Comment by kxbmap** [comment]
I cannot reproduce the problem.

There are no problem with TravisCI.
0.3.2: https://travis-ci.org/kxbmap/sbt-jooq/jobs/287880334#L555-L654
0.3.3: https://travis-ci.org/kxbmap/sbt-jooq/jobs/291565534#L557-L656

Could you run the following sbt commands and post the results?

> show libraryDependencies
> show JooqCodegen / fullClasspath
> show JooqCodegen / run / javaOptions

**Comment by w25r** [comment]
So, I upgraded to the latest version of the plugin (latest published, `0.4.1`) and I could still reproduce the issue.  I upgraded jOOQ and the problem still was there.  But then I upgraded the version of SBT to `1.3.4` (from `1.0.4`) and it went away... still looks a bit odd with logging doubled up:

```console
[info] [INFO] 
[info]                                       
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[info] @@@@@@@@@@@@@@@@  @@        @@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@
[info] @@@@@@@@@@@@@@@@  @@  @@    @@@@@@@@@@
[info] @@@@@@@@@@  @@@@  @@  @@    @@@@@@@@@@
[info] @@@@@@@@@@        @@        @@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[info] @@@@@@@@@@        @@        @@@@@@@@@@
[info] @@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
[info] @@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
[info] @@@@@@@@@@        @@  @  @  @@@@@@@@@@
[info] @@@@@@@@@@        @@        @@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@  @@@@@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
[info] @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  Thank you for using jOOQ 3.11.5
```

But the annoying part was the "error".  Thanks for looking into this, but I now see no reason to keep this open.

**Comment by kxbmap** [comment]
First `[info]` means that sbt showed stdout of the application.
Second `[INFO]` means that jOOQ logged messages as INFO level.

You can ommit sbt's `[info]` using the `outputStrategy` setting.

```scala
Jooq / run / outputStrategy := Some(StdoutOutput)
```
